testplan/testing/multitest/base.py

```diff
@@ -1310,6 +1313,47 @@ def _run_testsuite_iter(self, testsuite, testcases):
         if teardown_report is not None:
             yield teardown_report, [self.uid(), testsuite.uid()]
 
+    def _get_parent_uids(self, testsuite, testcase):
+        """
+        Utility method to get parent UIDs of a particular testcase.
+
+        :param testsuite: suite to which the case belongs
+        :param testcase: the testcase for which the UIDs are derived
+        :return: list of parent UIDs
+        """
+
+        param_template = getattr(
+            testcase, "_parametrization_template", None
+        )
+        if param_template:
+            parent_uids = [
+                self.uid(),
+                testsuite.uid(),
+                testcase._parametrization_template,
+            ]
+        else:
+            parent_uids = [self.uid(), testsuite.uid()]
+        return parent_uids
+
+    def _skip_testcases(self, testsuite, testcases):
```

Comment:
This may not be ideal, but functions as expected. We can think of a better name, simply add the necessary logic to the testcase runner method itself, or making this even more specialized.